docker-compose.yml

```diff
@@ -216,4 +219,7 @@ services:
       - redis
       - foundry
     networks:
-      - simulation
\ No newline at end of file
+      - simulation
+
+volumes:
```

Comment:
is this needed?

Reply:
The shared volume is required for different services to have access to read and write to the same data/object. It doesn't seem possible to have a variable shared between containers. It is the only way I could find for the nwaku containers to have access to the private keys generated by anvil. The alternative is to hardcode the private keys to a variable, like you suggested initially.